CMakeLists.txt

```diff
@@ -678,11 +678,22 @@ if(UNIX AND NOT APPLE)
 endif()
 
 
-if(NOT UNIX)
-  include(InstallRequiredSystemLibraries)
+##########################################################   S Y S T E M    L I B R A R I E S   ######################################################
+
+# Add compiler-provided system runtime libraries
+# TODO: Old code was just "NOT UNIX"
+if( WIN32 AND NOT UNIX )
```

Comment:
* [ ] TODO. Though now that I look at it, it seems `if (WIN32)` makes more sense...